libc/vga/tty.greg.c

```diff
@@ -380,8 +381,19 @@ static void TtyTextDrawChar(struct Tty *tty, size_t y, size_t x, wchar_t wc) {
   uint8_t attr = TtyGetTextAttr(tty);
   struct VgaTextCharCell *ccs = (struct VgaTextCharCell *)tty->canvas;
   size_t i = tty->y * Xn(tty) + tty->x;
-  int c = unbing(wc);
-  if (c == -1) c = 0xFE;
+  int c;
+  /*
+   * This might be an emergency console, so do not rely too much on unbing()
```

Comment:
If unbing() is becoming mission critical then its size optimization needs to go away. I've fixed this for you in https://github.com/jart/cosmopolitan/commit/29d695f4c92bfe5c33b26faf7467e3f513f63464

Reply:
Thanks @jart.  I have now removed the change to `tty.greg.c`.